web/src/reducers/budget.js

```diff
@@ -36,7 +36,7 @@ const initialState = years => ({
 
 const activities = src => Object.values(src.activities.byId);
 
-const addBudgetBlocks = (into, from) => {
+const addBudgetBlocks = (into, from = { total: 0, federal: 0, state: 0 }) => {
```

Comment:
Handles the case where an expense type (state personnel, contractor, non-personnel) doesn't have any rows.